common/classify.py

```diff
@@ -108,16 +118,17 @@ def test_and_report(self):
                 indx = data['email_index'].reshape(sample_size, 1)
                 test_mess_id = data['message_id'].reshape(sample_size, 1).astype("S200")
                 test_res = self.output_phish_probabilities(test_X, indx, root, test_indx, test_mess_id)
-                if test_res is not None:
+                if test_res != None:
                     results = np.concatenate((results, test_res), 0)
+            num_senders_completed += 1
         
         self.write_as_matfile(results)
         # Deletes message_id column, because no longer needed.
         results = np.delete(results, MESS_ID_IND, 1)
         res_sorted = results[results[:,PROBA_IND].argsort()][::-1]
         self.num_phish, self.test_size = self.calc_phish(res_sorted)
         output = self.filter_output(res_sorted)
-        progress_logger.info(pp.pformat(output))
+        progress_logger.info(pp.pprint(output))
```

Comment:
okay yeah and I had removed import logging at the top. other than that, now the terminal output matches that in progress.log. sorry about the setback, pushing now